### ISSUE TITLE
Make compatible with python 3.X

### ISSUE THREAD

**Issue by brando90** [pcattori/namespaces#9]
Make sure its compatible with python 3.

**Comment by brando90** [comment]
Not sure if this is a 3.X problem, but it seems namespaces can't find some utils module:

```
(venv2) brandomiranda~/Documents/MIT/MEng/hbf_tensorflow_code/tf_experiments_scripts $ python batch_main.py
Traceback (most recent call last):
  File "batch_main.py", line 12, in <module>
    import namespaces as ns
  File "/Users/brandomiranda/Documents/MIT/MEng/hbf_tensorflow_code/venv2/lib/python3.5/site-packages/namespaces/__init__.py", line 1, in <module>
    from namespaces.utils import *
ImportError: No module named 'namespaces.utils'
```

I don't think I had this issue with python 2.7

**Comment by brando90** [comment]
why are you importing utils?

https://github.com/pcattori/namespaces/blob/master/namespaces/__init__.py#L1

there isn't such a module.

**Comment by brando90** [comment]
Ok, I recently tried `import namespaces as ns` in my old venv with python 2.7.11 and it was able to import namespaces.

**Comment by brando90** [comment]
yes I have namespaces (1.0.0) vs namespaces (3.0.0). Ok I installed the old version and namespaces and I have no errors. It seems we just need to remove the import from __init__ python.

**Comment by pcattori** [comment]
acceptance criteria: tests are written in python3 compatible code and tests pass when running under python3

**Comment by pcattori** [comment]
blocked by pcattori/icicle#2

**Comment by pcattori** [comment]
closed by #24
